### PR TITLE
Adds functionality for sort and filter for team members specifically.

### PR DESCRIPTION
**Pull request by BobVerheij**

### CODE REVIEW

File: src/components/filter/FilterDropDown.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  content: any;
+  handleChange: Function;
+}
+
+function FilterDropDown({ id, labelValue, content, handleChange }: FilterDropDownProps) {
+  const options = new Set<string>();
+
+  content?.map((item) => options.add(item[id]));
+  const optionsArray = Array.from(options);
+  // console.log(optionsArray);
```
Comment (sabinewst) [inline src/components/filter/FilterDropDown.tsx:17]: Remove log

File: src/components/filter/FilterDropDown.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  labelValue: string;
+  content: any;
+  handleChange: Function;
+}
+
+function FilterDropDown({ id, labelValue, content, handleChange }: FilterDropDownProps) {
+  const options = new Set<string>();
+
+  content?.map((item) => options.add(item[id]));
+  const optionsArray = Array.from(options);
```
Comment (sabinewst) [inline src/components/filter/FilterDropDown.tsx:16]: Nice! Would be even better if you can filter out the undefined values :)

**Review by sabinewst — CHANGES_REQUESTED** [review]
Cool! Since this PR is for the team members page specifically, you can remove the filter bar from the time entries page for now. Also would be nice if you add some mobile styling :) 

Oh, and if you have some extra(!) time, try adding a message for when you're searching for a name and there are no results. Also something fun to try: allow searching for a combination of first and last name. But you can also skip this 😉

File: src/components/filter/FilterDropDown.tsx (hunk 2 above)
Comment (BobVerheij) [reply src/components/filter/FilterDropDown.tsx:16]: Good point!

**Pull request merged**